clang/lib/DPCT/GenMakefile.cpp

```diff
@@ -410,7 +410,7 @@ genMakefile(clang::tooling::RefactoringTool &Tool, StringRef OutRoot,
                                   "_FLAG_" + std::to_string(Idx);
         OS << buildString("$(", ObjStrName, "):$(", SrcStrName, ")\n");
 
-        // Only apply dpcpp compiler to the files which are originally built
+        // Only apply 'icpx' compiler to the files which are originally built
```

Comment:
Line 413,  the  meaning of the status is changed when using icpx.

Reply:
Will update in the next updated PR.

Reply:
Apply 'icpx -fycl' to compile the files which are originally built nvcc